README.md

```diff
@@ -1,2 +1,2 @@
-# chef-server-image cookbook
+# chef_server_image cookbook
```

Comment:
We should have examples of using the resource in the README, and in general the README should follow the usual guidelines for a cookbook readme and list attributes, lwrp's, etc., with examples.